src/App.css

```diff
@@ -88,20 +123,287 @@ h2 {
+  align-items: flex-start;
+  margin-left: 0.5vw;
+}
+
+.porcentage {
+  width: 5vw;
+  height: 3vw;
+  font-family: Montserrat, sans-serif;
+  font-size: 3vw;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+  margin: 0;
+  padding: 0;
+}
+
+.status {
+  width: 4.2vw;
+  margin: 0;
+  opacity: 0.5;
+  font-family: Montserrat, sans-serif;
+  font-size: 1.4vw;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.book-element3 {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: flex-start;
+  border-left: 1px #e8e8e8 solid;
+  height: 5.5vw;
+  padding-left: 8vw;
+  margin-top: 2.5vw;
+  margin-right: 7vw;
+}
+
+.current-chapter {
+  opacity: 0.5;
+  font-family: RobotoSlab, sans-serif;
   font-size: 1vw;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
 }
 
-.add-book-form {
+.book-chapter {
+  font-family: RobotoSlab, sans-serif;
+  font-size: 1.2vw;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: #121212;
+  margin-top: 0.3vw;
+}
+
+.update-button {
+  width: 12vw;
+  height: 2.5vw;
   margin-top: 1vw;
+  font-family: RobotoSlab, sans-serif;
+  font-size: 0.7vw;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: #fff;
+  border-radius: 3px;
+  background-color: #0290ff;
+  cursor: pointer;
+}
+
+.form-title {
+  margin-left: 6.188vw;
+  margin-top: 0.5vw;
+  font-family: Montserrat, sans-serif;
+  font-size: 1.25vw;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: #888;
+}
+
+.add-book-form {
+  width: 87.5vw;
+  height: 3vw;
+  margin-left: 6.188vw;
+  margin-top: 0.5vw;
   display: flex;
   flex-direction: row;
   justify-content: left;
   align-items: flex-start;
   margin-bottom: 1vw;
 }
 
-h3 {
-  margin-top: -0.15vw;
-  padding-right: 1vw;
+input {
+  height: 3vw;
+  margin-right: 1vw;
+  width: 20.5vw;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+  font-family: Montserrat, sans-serif;
+  font-size: 1vw;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.15px;
+  color: #c4c4c4;
+  padding-left: 1vw;
+}
+
+.add-book-button {
+  border-radius: 3px;
+  height: 3vw;
+  width: 11.5vw;
+  background-color: #0290ff;
+  font-family: RobotoSlab, sans-serif;
+  font-size: 0.813vw;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: #fff;
+  cursor: pointer;
```

Comment:
- [ ] Great job styling this section, however, you are missing the horizontal line before the form, please add it
- [ ] There is a black outline on the border of the `Add book` button, kindly remove it to fit the design, you can do this by giving the button a style of `border: none;`

### Your design
<img width="826" alt="Screenshot 2023-07-08 at 21 22 37" src="https://github.com/JuanDBta/bookstore-project/assets/64774936/5e3833ae-5e80-497c-be01-77df885ab2e2">

### Zeplin design

![Screenshot 2023-07-08 at 21 22 37 (2)](https://github.com/JuanDBta/bookstore-project/assets/64774936/db86a6b1-b61a-40ad-ad72-8240cef7f19a)